From: =?utf-8?q?Timo_R=C3=B6hling?= <roehling@debian.org>
Date: Tue, 19 Mar 2024 21:48:22 +0100
Subject: Ignore invalid value runtime warnings in tests

---
 pytest.ini | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/pytest.ini b/pytest.ini
index 7154264..38e69ec 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -11,8 +11,9 @@ filterwarnings =
     ignore:numpy.dtype size changed
     ignore:numpy.ufunc size changed
     ignore::UserWarning:cpuinfo,
-    ignore: divide by zero encountered in log
-    ignore: invalid value encountered in log
+    ignore:divide by zero encountered
+    ignore:invalid value encountered
+    ignore:shared memory stats
 # Matrix PendingDeprecationWarning.
     ignore:the matrix subclass is not
     ignore:Importing from numpy.matlib is
