--- a/numpy/distutils/tests/test_system_info.py
+++ b/numpy/distutils/tests/test_system_info.py
@@ -221,6 +221,7 @@ class TestSystemInfoReading:
         assert_equal(tsi.get_libs("libraries", [self._lib2]), [self._lib2])
 
     @pytest.mark.skipif(not HAVE_COMPILER, reason="Missing compiler")
+    @pytest.mark.skip(reason="Debian: disabled as we cant properly compile")
     def test_compile1(self):
         # Compile source and link the first source
         c = customized_ccompiler()
@@ -238,6 +239,7 @@ class TestSystemInfoReading:
     @pytest.mark.skipif(not HAVE_COMPILER, reason="Missing compiler")
     @pytest.mark.skipif('msvc' in repr(ccompiler.new_compiler()),
                          reason="Fails with MSVC compiler ")
+    @pytest.mark.skip(reason="Debian: disabled as we cant properly compile")
     def test_compile2(self):
         # Compile source and link the second source
         tsi = self.c_temp2
--- a/numpy/tests/test_reloading.py
+++ b/numpy/tests/test_reloading.py
@@ -5,6 +5,7 @@ import sys
 import subprocess
 import textwrap
 from importlib import reload
+import pytest
 
 
 def test_numpy_reloading():
@@ -35,6 +36,7 @@ def test_novalue():
                                           protocol=proto)) is np._NoValue)
 
 
+@pytest.mark.skip(reason="Debian: disabled it runs in a separate process, picking up the system-wide module")
 def test_full_reimport():
     """At the time of writing this, it is *not* truly supported, but
     apparently enough users rely on it, for it to be an annoying change
