--- a/setup.py
+++ b/setup.py
@@ -291,7 +291,7 @@ def parse_setuppy_commands():
     good_commands = ('develop', 'sdist', 'build', 'build_ext', 'build_py',
                      'build_clib', 'build_scripts', 'bdist_wheel', 'bdist_rpm',
                      'bdist_wininst', 'bdist_msi', 'bdist_mpkg', 'build_src',
-                     'bdist_egg')
+                     'bdist_egg', 'config')
 
     for command in good_commands:
         if command in args:
